README.md

```diff
@@ -37,6 +37,7 @@ def test_click_light_turn_on_for_5_minutes(given_that, living_room, assert_that)
   * [Initial Setup](#initial-setup)
   * [Write you first unit test](#write-you-first-unit-test)
   * [Result](#result)
+  * [Deprecation Warnings](#deprecation-warnings)
```

Comment:
Could we put it under `Advanced Usage` ? 

I'd like to not clutter the `Quick guide` with mention of deprecation warnings.

Reply:
Sounds good to me.